docs/01-intro.md

```diff
@@ -6,63 +6,72 @@ sidebar_position: 1
 
 # Introduction
 
-:::info
-CosmWasm is a new smart contracting platform built for the Cosmos ecosystem. If you haven't yet heard
-of it, please [check out this intro](https://blog.cosmos.network/announcing-the-launch-of-cosmwasm-cc426ab88e12). The
-purpose of this documentation is to give a deep dive into the technology for developers who wish to try it out or
-integrate it into their product. Particularly, it is aimed at Go developers with experience with the Cosmos SDK, as well
+:::caution
+This documentation is currently being refactored. Some parts may be outdated or broken.
+
+Feel free to contribute through [GitHub repo](https://github.com/InterWasm/docs) or contact us through the **cw-docs-work-group** channel in the [CosmWasm discord](https://discord.gg/ksZw5ReW)
+:::
+
+## What is CosmWasm?
+
+CosmWasm is a smart contracting platform built for the Cosmos ecosystem. Simply put, it's the Cosmos (Cosm) way of using WebAssembly (Wasm) hence the name.  
+
+CosmWasm is written as a module that can plug into the Cosmos SDK. This means that anyone currently building a blockchain using the Cosmos SDK can quickly and easily add CosmWasm smart contracting support to their chain, without adjusting existing logic.
+
+Rust is the most heavily used programming language for CosmWasm but any language that can compile into WebAssembly can be used if supported.
```

Comment:
The sentence feels like it is possible to implement cw contracts with other languages. 
Best to write something like:
Rust is currently used programming language for CosmWasm, in future it is possible to have different pls like AssemblyScript